public_html/wp-content/mu-plugins/load-other-mu-plugins.php

```diff
@@ -10,9 +10,8 @@
  * Load mu-plugins that should run on all networks.
  */
 function wcorg_include_common_plugins() {
-	require_once dirname( __DIR__ ) . '/mu-plugins-private/wporg-mu-plugins.php';
-
 	if ( 'local' !== WORDCAMP_ENVIRONMENT ) {
+		require_once dirname( __DIR__ ) . '/mu-plugins-private/wporg-mu-plugins.php';
```

Comment:
Moving it here could have some minor side effects. Nothing huge, but it'd be ideal to keep it above, and add an `if ( file_exists(  ...` condition.